scripts/generate_share_images/index.ts

```diff
@@ -10,48 +10,76 @@ import fs from 'fs-extra';
 import path from 'path';
 import Pageres from 'pageres';
 import urlJoin from 'url-join';
-import { STATES } from '../../src/common';
-
-// We don't care about the values here, but this is a cheap way to determine all
-// of the counties we have any data for and should therefore screenshot.
-import CalculatedCountyInterventionJSON from '../../src/assets/data/calculated_county_interventions.json';
-const COUNTIES = Object.keys(CalculatedCountyInterventionJSON);
```

Comment:
NOTE: I switched to using the actual projections to figure out what states/counties we have data for so that I can do it at a per-chart level, since many (most) counties don't have all the different charts.